src/dotnet/Chat.UI.Blazor/Components/ContactNavItem.razor

```diff
@@ -68,4 +68,5 @@
             isRecording);
     }
 
+    public record Model(bool IsOnline, string ActiveChatId, string FullChatId, string ShortChatId, bool IsListening, bool IsRecording);
```

Comment:
Let's split this to lines.

Reply:
One more thing: let's also order this stuff + do we really need "ActiveChatId" here? I.e. why not `IsActive`?

Reply:
All in all, let's make this readable, logically structured, + w/o unnecessary data.